dataprofiler/tests/profilers/test_text_column_profile.py

```diff
@@ -607,3 +610,211 @@ def test_diff(self):
             places=2,
         )
         self.assertDictEqual(expected_diff, profile_diff)
+
+    @mock.patch("time.time", return_value=0.0)
+    def test_json_encode_after_update(self, time):
+        self.maxDiff = None
+        df = pd.Series(
+            [
+                "abcd",
+                "aa",
+                "abcd",
+                "lito-potamus",
+                "b",
+                "4",
+                ".098",
+                "2",
+                "dfd",
+                "2",
+                "12.32",
+            ]
+        ).apply(str)
+
+        text_options = TextOptions()
+        text_options.histogram_and_quantiles.bin_count_or_method = 5
+        profiler = TextColumn(df.name, text_options)
+        with test_utils.mock_timeit():
+            profiler.update(df)
+
+        serialized_dict = json.loads(json.dumps(profiler, cls=ProfileEncoder))
+
+        # popping quantiles and comparing as list below since it is so large
+        serialized_quantiles = serialized_dict["data"].pop("quantiles")
+
+        # popping vocab and comparing as set below since order is random
+        serialized_vocab = serialized_dict["data"].pop("vocab")
+
+        serialized = json.dumps(serialized_dict)
+
```

Comment:
add this section so we can test `quantiles` and `vocab`. I use the `profile.quantiles` and `profiler.vocab` as my expected which I think is fair.